test/integration/Makefile

```diff
@@ -56,7 +56,7 @@ deploy-ccm:
 	kubectl -n kube-system create -f secrets.yaml && \
 	kubectl -n kube-system apply -f ../../manifests/controller-manager/cloud-controller-manager-roles.yaml && \
 	kubectl -n kube-system apply -f ../../manifests/controller-manager/cloud-controller-manager-role-bindings.yaml && \
-	sed 's~gcr.io/cloud-provider-vsphere/cpi/release/manager:latest~$(CCM_IMAGE)~g' <../../manifests/controller-manager/vsphere-cloud-controller-manager-pod.yaml | kubectl -n kube-system apply -f -
+	sed 's~gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.2.1~$(CCM_IMAGE)~g' <../../manifests/controller-manager/vsphere-cloud-controller-manager-pod.yaml | kubectl -n kube-system apply -f -
```

Comment:
This seems fine to unblock PRs, but I think ideally we just create a separate file for integration tests